src/main/java/teammates/ui/controller/Action.java

```diff
@@ -576,10 +584,131 @@ private boolean isMasqueradeModeRequested(AccountAttributes loggedInUser, String
                && loggedInUser.googleId != null
                && !loggedInUser.googleId.equals(requestedUserId);
     }
+
+    /**
+     *Common method to Get the feedback data
+     */
+
+    protected FeedbackSessionAttributes extractFeedbackSessionData() {
+
+        FeedbackSessionAttributes newSession = new FeedbackSessionAttributes();
+
+        newSession.setCourseId(getRequestParamValue(Const.ParamsNames.COURSE_ID));
+        newSession.setFeedbackSessionName(Sanitizer.sanitizeTitle(
+                getRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_NAME)));
+
+        // getting the class name from which it's being called
+        String className = this.getClass().getName();
+
+        if ("InstructorFeedbackEditSaveAction".equals(className)) {
```

Comment:
@thenaesh Thanks for the help. Finally `this.doSomething()` cleared my all confusions.